nodes/scheduling/nodes-scheduler-about.adoc

```diff
@@ -5,42 +5,47 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-
-
 Pod scheduling is an internal process that determines placement of new
 pods onto nodes within the cluster.
 
 The scheduler code has a clean separation that watches new pods
 as they get created and identifies the most suitable node to host them. It then
 creates bindings (pod to node bindings) for the pods using the master API.
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
-
 Default pod scheduling::
-{product-title} comes with a xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[default scheduler] that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+{product-title} comes with a default scheduler that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+
+////
+// TODO: Determine if we want to pull in any info about the default scheduler from https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html
+
+The default OpenShift Container Platform pod scheduler is responsible for determining placement of new pods onto nodes within the cluster. It reads data from the pod and tries to find a node that is a good fit based on configured policies. It is completely independent and exists as a standalone/pluggable solution. It does not modify the pod and just creates a binding for the pod that ties the pod to the particular node.
+
+and
+
+https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-about_nodes-scheduler-default
+////
 
 Advanced pod scheduling::
-In situations where you might want more control over where new pods are placed, the {product-title} advanced scheduling features allow you to configure a pod so that the pod is required or has a preference to run on a particular node or alongside a specific pod by.
+In situations where you might want more control over where new pods are placed, the {product-title} advanced scheduling features allow you to configure a pod so that the pod is required or has a preference to run on a particular node or alongside a specific pod.
```

Comment:
This sentence is longer than ISG's recommended 25 words. Also, I think this goes against the recommended usage of _allow_. Does it make sense to rewrite? 

Perhaps: "You can use the {product-title} advanced scheduling features if you want more control over the placement of new pods. You can configure the affinity of a pod to make it required, define a preference for a pod to run on or alongside a particular node." 

I'm not happy with that rewording of the second sentence. To me, it does not clarify the links in the xrefs below. I wonder if it is necessary?